tests/ltp/patches/ltp_setfsuid_setfsuid03.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/testcases/kernel/syscalls/setfsuid/setfsuid03.c b/testcases/kernel/syscalls/setfsuid/setfsuid03.c
+index 0e5f860c8..9002af8ba 100644
+--- a/testcases/kernel/syscalls/setfsuid/setfsuid03.c
++++ b/testcases/kernel/syscalls/setfsuid/setfsuid03.c
+@@ -22,7 +22,7 @@
+  * Testcase to test the basic functionality of the setfsuid(2) system
+  * call when called by a user other than root.
+  */
+-
++// Patch Description: Test is failing to open /etc/passwd file with nobody user, as test intention is not testing /etc/passwd, moved this piece of code to setup to get uid
```

Comment:
Can you keep a reasonable line length? Good places to break with new line are commas and full stops.